### PR TITLE
[TS] LPS-154557 Add config to allow TunnelServlet to send exceptions again

### PR DESCRIPTION
**Pull request by vendeltoreki**
Hi @liferay-core-infra , @dantewang ,

I'm resending my previous PR: https://github.com/liferay-core-infra/liferay-portal/pull/878
Including the commits added in https://github.com/vendeltoreki/liferay-portal/pull/243

I've also included an integration test. It ran successfully before and after the SF changes as well.

Please review the changes.

Thanks,
Vendel

### CODE REVIEW

**Review by dantewang — APPROVED** [review]

**Comment by dantewang** [comment]
@tinatian Code looks good to me.

@zhangyan0701 Could you review the test results? Thanks!

**Comment by zhangyan0701** [comment]
No regressions found.

File: modules/apps/staging/staging-test/src/testIntegration/java/com/liferay/staging/test/StagingImplTest.java [hunk 1]
```diff
@@ -268,6 +272,36 @@ public void testRemoteStaging() throws Exception {
 		enableRemoteStaging(false);
 	}
 
+	@Test
+	public void testRemoteStagingHiddenError() throws Exception {
+		Throwable caughtThrowable = _enableRemoteStagingWithError();
```
Comment (tinatian) [inline modules/apps/staging/staging-test/src/testIntegration/java/com/liferay/staging/test/StagingImplTest.java:277]: @vendeltoreki Please also set the property explicitly to make sure this integration test not depending on your current setting.

**Comment by tinatian** [comment]
@vendeltoreki Please see my comments and resend.

Thanks.
Tina

**Pull request closed without merging**